src/main/java/games/strategy/engine/framework/startup/ui/LocalSetupPanel.java

```diff
@@ -33,74 +27,26 @@
   public LocalSetupPanel(final GameSelectorModel model) {
     m_gameSelectorModel = model;
     createComponents();
-    layoutComponents();
+    layoutPlayerComponents(this, m_playerTypes, m_gameSelectorModel.getGameData());
     setupListeners();
     setWidgetActivation();
   }
 
   private void createComponents() {}
```

Comment:
This private no-op method can probably be removed unless there are plans to pull it into `SetupPanel` as a potentially overridable method.